add-ons/fabric8/fabric8.addon

```diff
@@ -0,0 +1,37 @@
+# Name: fabric8
+# Description: Installs fabric8 locally on Minishift
+# Required-Vars: GITHUB_OAUTH_CLIENT_ID,GITHUB_OAUTH_CLIENT_SECRET
```

Comment:
How does one get to the values for these variables? What do I need to setup/configure?